packages/widgets/src/components/NotificationCard/NotificationCard.tsx

```diff
@@ -0,0 +1,67 @@
+import { useTheme } from '@emotion/react';
+import { Badge, CardBase, Icon, Indicator, Typography } from '@ssa-ui-kit/core';
+import {
+  notifyBadge,
+  notifyCard,
+  notifyTitle,
+  notifyText,
+  notifyTime,
+} from './style';
```

Comment:
Let's call this file `./style**s**` - I see we use the plural here. There are two files only that call it `style.ts`. But we'll rename them eventually.

Reply:
done